src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -158,6 +193,10 @@ export default {
 	&__input {
 		width: 100%;
 	}
+
+	dd {
```

Comment:
Please always target elements with classes :) 

https://vue-loader.vuejs.org/guide/scoped-css.html#also-keep-in-mind